src/containers/pages/MDALite/cddReports/cdds.tsx

```diff
@@ -0,0 +1,208 @@
+  service: typeof supersetFetch;
+  wardData: MDALiteWards[];
+}
+
+/** Component for displaying MDA Lite CDD Reports */
+const MDALiteCddReports = (props: MDALiteCddReportsProps & RouteComponentProps<RouteParams>) => {
+  const { cddData, wardData, service, fetchCDDs, fetchWards } = props;
+
+  const [loading, setLoading] = useState<boolean>(false);
+
+  const { params } = props.match;
+  const { planId, jurisdictionId, supervisorId } = params;
+  const supervisorName = (cddData[0]?.supervisor_name || supervisorId) as string;
+
+  async function loadData() {
+    setLoading(cddData.length < 1);
+    try {
+      const planIdFilter: SupersetAdhocFilterOption[] = [];
+      const jurisdictionFilter: SupersetAdhocFilterOption[] = [];
+      const supervisorFilter: SupersetAdhocFilterOption[] = [];
+      if (jurisdictionId && supervisorId && planId) {
+        jurisdictionFilter.push({
+          comparator: jurisdictionId,
+          operator: '==',
+          subject: 'base_entity_id',
+        });
+        planIdFilter.push({ comparator: planId, operator: '==', subject: 'plan_id' });
+        supervisorFilter.push({
+          comparator: supervisorId,
+          operator: '==',
+          subject: 'supervisor_id',
+        });
+        // get cdd data
+        const fetchCDDParams = superset.getFormData(SUPERSET_MAX_RECORDS, [
+          ...planIdFilter,
+          ...jurisdictionFilter,
+          ...supervisorFilter,
+        ]);
+        await service(SUPERSET_MDA_LITE_REPORTING_CDD_DATA_SLICE, fetchCDDParams).then(res =>
+          fetchCDDs(res)
+        );
+        // get ward data
+        const fetchWardParams = superset.getFormData(SUPERSET_MAX_RECORDS, [
+          ...planIdFilter,
+          ...jurisdictionFilter,
+        ]);
+        if (!wardData.length) {
+          await service(SUPERSET_MDA_LITE_REPORTING_WARD_SLICE, fetchWardParams).then(res =>
+            fetchWards(res)
+          );
+        }
+      }
+    } catch (e) {
+      displayError(e);
+    } finally {
+      setLoading(false);
+    }
+  }
+
+  useEffect(() => {
+    loadData().catch(err => displayError(err));
+  }, [planId, jurisdictionId, supervisorId]);
+
+  const currentPage = {
+    label: supervisorName,
+    url: `${REPORT_MDA_LITE_CDD_REPORT_URL}/${planId}/${jurisdictionId}/${supervisorId}`,
+  };
+  const breadcrumbProps = {
+    currentPage,
+    pages: [
+      {
+        label: '...',
+        url: `${REPORT_MDA_LITE_WARD_URL}/${planId}/${wardData[0]?.parent_id}`,
+      },
+      {
+        label: wardData[0]?.ward_name || '...',
+        url: `${REPORT_MDA_LITE_CDD_REPORT_URL}/${planId}/${jurisdictionId}`,
+      },
+    ],
+  };
+  const currentTitle = supervisorName
+    ? `${MDA_LITE_REPORTING_TITLE}: ${supervisorName}`
+    : MDA_LITE_REPORTING_TITLE;
+
+  // table props
+  const tableProps = getCddTableProps(cddReportColumns, cddData, props);
+
+  if (loading) {
+    return <Loading />;
+  }
+  return (
+    <div>
```

Comment:
Would it be better to use React fragment `<>...</>` in this particular case instead of `<div>....</div>`